pkgs/applications/misc/blugon/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3, libX11, libXrandr, gcc, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "blugon";
+  version = "1.11.4";
+
+  src = fetchFromGitHub {
+    owner = "jumper149";
+    repo = "blugon";
+    rev = "c53b47eac7f9b23adeb62f8df10f38ffbc0dd156";
```

Comment:
Please tag a release on GitHub and refer to it here.

Reply:
Created a [release](https://github.com/jumper149/blugon/tree/1.11.4).
Commit hash stays the same.